aten/src/ATen/native/native_functions.yaml

```diff
@@ -5301,6 +5301,12 @@
     CUDA: cat_out_cuda
     QuantizedCPU: cat_out_quantized_cpu
 
+- func: _foreach_add.Scalar(Tensor[] tensors, Scalar scalar) -> Tensor[]
```

Comment:
@ezyang recommends disabling device_guard codegen here by setting `device_guard: False` and set device guard manually in the fast cuda implementation.